### DISCUSSION TITLE
Team's Privacy - Future Sprints ?

### DISCUSSION THREAD

**Discussion by rkrishna-sdge** [microsoft/vsts-extension-retrospectives#474] [General]
I see the retrospectives conducted within a team / set of teams are open and visible to other teams (same project) who do not participate in the retrospective. Is there a way to make it private ( we try to make this a safe space for team(s) within the retro) ?

**Comment by dstoddar** [comment]
This capability is on the roadmap but we don't have a timeframe yet.

**Reply by rkrishna-sdge** [reply]
@dstoddar - Do you have this capability for [team's privacy](https://github.com/microsoft/vsts-extension-retrospectives/discussions/474#discussion-4503013) in the road map ?

**Comment by dphanson63** [comment]
Permissions has been implemented and has been working.
